Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/site/parser/CommentParser.java

```diff
@@ -156,6 +171,16 @@ private CharSequence handleAnchor(
                 int postNo = (int) handlerLink.value;
                 post.addReplyTo(postNo);
 
+                MockReplyManager.MockReply mockReply = mockReplyManager.getLastMockReplyOrNull(
+                        post.board.siteId,
+                        post.board.code,
+                        post.opId
+                );
+
+                if (mockReply != null) {
+                    addMockReply(theme, post, spannableStringBuilder, postNo, mockReply);
```

Comment:
Rather than doing this whole thing here, which it looks like it would need to be attached to another quote for it to even be created, so no mocking with posts without existing quotes in them, it might be better to do this in DefaultPostParser, in parseComment, right before the TextUtils.concat call.

Reply:
>which it looks like it would need to be attached to another quote for it to even be created

No, I just add a new quote to the repliesToIds set. There not necessary have to be other cause in a post already to add a mock reply to it. And you are suggesting to add a quote directory to the HTML which I think is not really necessary because I can just operate on a level of already parsed posts which I think is a less hacky way to add a new quote to a post.